submissions/Eugene-CG/OOP-exercise/index.js

```diff
@@ -0,0 +1,99 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+   Code repository: https://github.com/Eugene-CG/a-tiny-JS-world
+   Web app: https://eugene-cg.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Inhabitant {
+  constructor(name, saying, gender) {
+    this.name = name;
+    this.saying = saying;
+    this.gender = gender;
+  }
+  getProperties() {
+    return `species:${this.species}; name:${this.name}; gender:${this.gender}; legs:${this.legs}; hands:${this.hands}; saying:${this.saying};`;
+  }
+}
+class Animal extends Inhabitant {
+  constructor(name, saying, gender) {
+    super(name, saying, gender);
+    this.paws = 4;
+    this.species = "animal";
+  }
+  getProperties() {
+    return `species:${this.species}; name:${this.name}; gender:${this.gender}; paws:${this.paws}; saying:${this.saying};`;
```

Comment:
I believe the `Inhabitant` class which is being extended here already has `species, name` etc in its saying.